### PR TITLE
feat(CSN-856): burn-miner-emissions

### PR DESCRIPTION
**Pull request by nauttiilus**
- Burn Mechanism: Implements a burn-only weight setting mechanism.
- Reintroduces scoring for low-end GPUs.

### CODE REVIEW

**Review by IvanAnishchuk — APPROVED** [review]
I'd probably go for something even simpler but this looks functional. Approved.

File: neurons/validator.py [hunk 1]
```diff
@@ -1054,6 +1054,56 @@ async def deallocate_miner(self, axon, public_key):
...
+        """
+        # 1) fetch burn UID
+        burn_uid = self.get_burn_uid()
+
+        # 2) prepare a single-element score tensor
+        scores = torch.tensor([1.0], dtype=torch.float32)
+        scores[scores < 0] = 0
+
+        # 3) normalize into a weight vector that sums to 1
+        weights: torch.FloatTensor = torch.nn.functional.normalize(scores, p=1.0, dim=0).float()
```
[the remark this answers is not in the document]
Comment (nauttiilus) [reply neurons/validator.py:1089]: I started with a parameter to set the burn ratio but skipped it for now, which would have required normalization. Negative scores can't happen anymore since the PoG, so we could skip this in the future. Agreed.
